doc/source/whatsnew/v1.0.0.rst

```diff
@@ -712,7 +712,7 @@ or ``matplotlib.Axes.plot``. See :ref:`plotting.formatters` for more.
 - Removed :meth:`DataFrame.from_items` (:issue:`18458`)
 - Removed :meth:`DataFrame.as_matrix`, :meth:`Series.as_matrix` (:issue:`18458`)
 - Removed :meth:`Series.asobject` (:issue:`18477`)
-- Removed :meth:`DataFrame.as_blocks`, :meth:`Series.as_blocks`, `DataFrame.blocks`, :meth:`Series.blocks` (:issue:`17656`)
+- Removed :meth:`DataFrame.as_blocks`, :meth:`Series.as_blocks`, :attr:`DataFrame.blocks`, :attr:`Series.blocks` (:issue:`17656`)
```

Comment:
i don't think these work on removed functions / attrivutes

Reply:
Yeah, they don't.  This is true for most of the references in this section.

Reply:
yep, might as well remove the references (but prob doesn't hurt much either)

Reply:
okay, went through the entire section and deleted the references from from things that have been removed